sdk/servicebus/service-bus/test/stress/stressTestsBase.ts

```diff
@@ -428,37 +489,48 @@ export class SBStressTestsBase {
   }
 
   public async end() {
-    await this.snapshot();
-
-    if (this.snapshotOptions.snapshotFocus.includes("receive-info")) {
-      const output = await saveDiscrepanciesFromTrackedMessages(this.trackedMessageIds);
+    try {
+      await this.snapshot();
+
+      if (this.snapshotOptions.snapshotFocus?.includes("receive-info")) {
+        const output = await saveDiscrepanciesFromTrackedMessages(this.trackedMessageIds);
+
+        defaultClient.trackEvent({
+          name: "discrepencies",
+          properties: {
+            messages_sent_but_never_received: output.messages_sent_but_never_received.join(","),
+            messages_not_sent_but_received: output.messages_not_sent_but_received.join(","),
+            messages_sent_multiple_times: output.messages_sent_multiple_times.join(","),
+            messages_sent_once_but_received_multiple_times: output.messages_sent_once_but_received_multiple_times.join(
+              ","
+            ),
+            messages_sent_once_and_received_once: output.messages_sent_once_and_received_once.join(
+              ","
+            )
```

Comment:
Maybe, this can be removed.. since it doesn't sound like a discrepancy and would be an unneeded noise.